### DISCUSSION TITLE
How to enable bridged network on tart virtual machines?

### DISCUSSION THREAD

**Discussion by cosmoshepcat** [cirruslabs/tart#244] [Q&A] [answered]
Is there a cli option or setting somewhere to enable bridged mode networking on the running virtual machine so that it completely shares the host machines network and is reachable directly?

You can do this in products like parallels via configuration file and it will fire up and grab a DHCP ip from the same network as the host machine.  We can then install other utilities etc.

**Comment by fkorotkov** [comment, marked as answer]
We just got necessary permissions from Apple and it will be available soon. Please follow #243 for  updates.

**Comment by cosmoshepcat** [comment]
looks like the support got merged in but the entitlements reverted a short bit later?  i did see the new command come through in latest pull of tart but got an error about entitlements when i tried it just to see.
